src/components/core/Page/GlobalNavbar/GlobalNavbar.tsx

```diff
@@ -22,6 +22,7 @@ const StyledGlobalNavbar = styled.div`
   display: flex;
   justify-content: flex-end;
 
+  position: relative;
```

Comment:
what's this for?
I see the commit message you added but I don't fully understand 😅
 `position GlobalNavBar and StyledBacklink so that collection navigation arrows cannot be clicked while in header`

Reply:
Before adding `position: relative`, the `  z-index: 1;` applied to `StyledGlobalNavbar` didn't do anything. `position: relative` creates a stacking context.

Because `z-index` wasn't applied to the navbar, that meant we could hover over "Back to Gallery" or "Sign In" (or whatever else is directly above Prev/Next arrows and unintentionally trigger the hover event on the arrows:

Here, I'm hovering over sign in but its instead registering a hover event on the arrow below:

<img width="117" alt="image" src="https://user-images.githubusercontent.com/13339581/156209019-5ef42c20-eecd-4b29-acfe-f6b9d20b7bc4.png">
 
The `position: relative` is just forcing the intended stacking context we wanted via `z-index: 1`